### ISSUE TITLE
Unable to Enter Time

### ISSUE THREAD

**Issue by cryreis** [leviFrosty/witness-work#162]
**Describe the bug**
I am no longer able to tap on the month to access previously entered time or to add new time. When I tap on the month it doesn’t do anything. I have restarted and updated my phone, uninstalled and reinstalled the app but the problem continues.

**To Reproduce**

**Expected behavior**
I would like to be able to view my hours for the month and add new entries 

**Screenshots**
If applicable, add screenshots to help explain your problem. (Draw on images if possible)

**Smartphone (please complete the following information):**

- Device: iPhone 12 Pro
- OS: iOS 17.6.1
- Version 17.6.1

**Additional context**

**Comment by leviFrosty** [comment]
@cryreis 

This isn't good! I apologize for you having an issue like this.

Can you help me help you by messagin me on Whatsapp / Signal: +15023348618

I'll try to get you a resolution quickly.

Yours,
Levi

**Comment by leviFrosty** [comment]
In the meantime, if you are busy:

You likely are set to Publisher type: 'Publisher'.

Publisher's don't have access to record 'Time' because it doesn't apply to them.

However, if you would like to record time for whatever personal reasons, you can work around this by changing your publisher type. Go to `Settings > Preferences > Publisher > Status`

Change it from: `Publisher`
To: `Custom`

This should solve your issue, please let me know if it doesn't. Or message me on Whatsapp/Signal.

Yours,
Levi

**Comment by cryreis** [comment]
Ah! That makes sense. I changed my publisher type and now I can see my
entries for the month. Problem solved! Thanks for the help troubleshooting.

May I suggest that a standard publisher still be able to record their time
though? I know for myself personally, I like to keep track of my hours so I
can set and reach personal goals even though we don’t have to report hours
any longer. This month specifically I was asked to track my hours by my
congregation elders because I applied to start regular pioneering in
September.

Crystal Reiszner
